Broken model run tests

I think the gmake file wasn't updated somewhere along the way, so the new NGen.cpp fails to build. 

```
[ 97%] Building CXX object CMakeFiles/ngen.dir/src/NGen.cpp.o
714
/ngen/src/NGen.cpp: In function 'int main(int, char**)':
715
/ngen/src/NGen.cpp:241:55: error: 'time_step' was not declared in this scope
716
         catchment_outfiles[formulation_pair.first] << time_step << "," << current_timestamp << "," << output_str << std::endl;
717
                                                       ^~~~~~~~~
718
/ngen/src/NGen.cpp:241:55: note: suggested alternative: 'time_step_t'
719
         catchment_outfiles[formulation_pair.first] << time_step << "," << current_timestamp << "," << output_str << std::endl;
720
                                                       ^~~~~~~~~
721
                                                       time_step_t
722
gmake[3]: *** [CMakeFiles/ngen.dir/build.make:63: CMakeFiles/ngen.dir/src/NGen.cpp.o] Error 1
723
gmake[2]: *** [CMakeFiles/Makefile2:75: CMakeFiles/ngen.dir/all] Error 2
724
gmake[1]: *** [CMakeFiles/Makefile2:87: CMakeFiles/ngen.dir/rule] Error 2
725
gmake: *** [Makefile:175: ngen] Error 2
726
The command '/bin/sh -c cmake --build /ngen --target ngen' returned a non-zero code: 2
727
```

## Current behavior
Model runs are failing at the build, and need to be fixed.

## Expected behavior
Every time the action is triggered with new PRs the model should run

## Steps to replicate behavior (include URLs)

1. https://github.com/NOAA-OWP/ngen/runs/1299970026?check_suite_focus=true


## Screenshots
<img width="1674" alt="Screen Shot 2020-10-26 at 7 52 49 AM" src="https://user-images.githubusercontent.com/60660437/97174596-4eb95480-1760-11eb-9d3b-96f502bddb94.png">

--- a/src/NGen.cpp
+++ b/src/NGen.cpp
@@ -238,7 +238,7 @@ int main(int argc, char *argv[]) {
             catchment_outfiles[formulation_pair.first] << "Time Step," << "Time," << header_str <<std::endl;
         }
         std::string output_str = formulation_pair.second->get_output_line_for_timestep(output_time_index);
-        catchment_outfiles[formulation_pair.first] << time_step << "," << current_timestamp << "," << output_str << std::endl;
+        catchment_outfiles[formulation_pair.first] << output_time_index << "," << current_timestamp << "," << output_str << std::endl;
         response = response * boost::geometry::area(nexus_collection->get_feature(formulation_pair.first)->geometry<geojson::multipolygon_t>());
         std::cout << "\t\tThe modified response is: " << response << std::endl;
         //update the nexus with this flow

